.github/workflows/main.yml

```diff
@@ -13,14 +13,14 @@ jobs:
     - name: Install Dependencies (Linux)
       run: |
         sudo apt-get update -y
-        sudo apt-get install -y devscripts debhelper moreutils fakeroot jq pigz help2man
+        sudo apt-get install -y devscripts debhelper moreutils fakeroot jq pigz help2man wget
         wget "https://github.com/koalaman/shellcheck/releases/download/latest/shellcheck-latest.linux.x86_64.tar.xz"
```

Comment:
wouldn't it make more sense to just use `curl`?

Reply:
I agree that we should avoid adding another package here

Reply:
Do note that both curl and wget are preinstalled for all supported action runners...

I grew up w/ `wget`, but unless one is doing spidering, `curl` is probably the right choice.

It doesn't really matter to me.

Reply:
During the testing I could not make curl work due to unexpected behaviour. Later I realised a fake curl was being triggered instead of the actual binary (https://github.com/github/backup-utils/blob/master/test/bin/curl) . As @jsoref mentioned it is not really a new dependency, therefore I would leave it as it is 🙂